Source/Krypton Components/Krypton.Toolkit/General/KryptonInputBoxData.cs

```diff
@@ -14,7 +14,7 @@ public struct KryptonInputBoxData
     {
         #region Public
 
-        public bool? UseRtlReading { get; set; }
+        public KryptonUseRTLLayout? UseRTLLayout { get; set; }
```

Comment:
Why is this nullable? it is initialised to `No`

Reply:
Removed nullable + add documentation